image.md

```diff
@@ -0,0 +1,308 @@
+The contents of this directory MUST appear to be materialized in such a way as to contain all of the files from the image, without the whiteout (deleted) files, and its dependent images.
+The exact mechanism, process and order, by which those files are materialized on disk is an implementation detail, however in the end it MUST appear as though all of the files from the images were written to disk in the order they were specified in the `layers` property.
+This means that each dependent image's `whiteout` property MUST appear to be processed during the materialization of that image so that if a subsequent image creates a file by the same name as one mentioned in the `whiteout` property then it is present at the end of the process.
+
+If a dependent image is not present in the image, and the implementation is unable to locate the referenced image by its name or ID, then an error MUST be generated.
+
+The image, and dependent images, MUST be verified by calculating the hash/ID of the materialized (uncompressed) tar file on disk and comparing that value with the ID from the image.
+If they do not match then an error MUST be generated.
+
+This specification does not mandate any requirements on the processing of additional files that might appear in the image.
+In other words, implementations are not required to materialize them on disk in the filesystem bundle.
+
+This specification does not mandate any ordering to when each piece of information from the image is materialized on disk.
+
+## Misc Notes
+
+There is no requirement that importing an image into a compliant OCI implementation and then exporting it will result in the same image.
+Nor is there any requirement that an OCI compliant implementation retain the original layering during such an import/export process.
+
+## Examples
+
+In the following examples, the contents of some of the files within the images are included (in abbreviated form) for clarity.
+
+### Example 1 -  An image with one layer:
+```
+SHA512-ABC.tar:
+|-- manifest.json
+|   |==  {
+|   |==    ...config.json properties...
+|   |==    "name": "myApp",
+|   |==    "root": {
+|   |==      "path": "rootfs"
+|   |==    },
+|   |==    "layers": [
+|   |==      { "name": "myapp",
+|   |==        "ID": "sha512-DDD" }
+|   |==    ]
+|   |==  }
+|
+|-- sha512-DDD.tar
+|   |-- manifest.json
+|   |   |== {
+|   |   |==   ...config.json properties...
+|   |   |==   "name": "myapp",
+|   |   |==   "root": {
+|   |   |==     "path": "myRoot"
+|   |   |==   }
+|   |   |== }
+|   |
+|   |-- myRoot/
+|   |   |-- home/jeff/runit
+```
+
+
+### Example 2 - An image with 4 layers, 2 of which are embedded:
+```
+sha512-DEF.tar:
+|-- manifest.json
+|   |== {
+|   |==   ...config.json properties...
+|   |==   "name": "myImage",
+|   |==   "root": {
+|   |==     "path": "myfs"
+|   |==   },
+|   |==   "layers": [
+|   |==     { "name": "example.com/ubuntu" },
```

Comment:
Someone want to propose some text?  We need to be a little careful because I don't necessarily want to get into defining "discovery" within this doc, but providing a statement saying that DNS is a good example of how an implementation can resolve it might be good.